src/main/java/in/twizmwaz/cardinal/module/modules/armorKeep/ArmorKeep.java

```diff
@@ -71,6 +71,7 @@ public void onPlayerDeath(PlayerDeathEvent event) {
 
     @EventHandler(priority = EventPriority.HIGHEST)
     public void onPgmSpawn(CardinalSpawnEvent event) {
+        if (event.isCancelled()) return;
```

Comment:
I believe all of these can be replaced with ignoreCancelled = true in the annotation